File: problem_5.py
def solution_5():
    number = 10
    while True:
        number += 1
        if (number % 11 == 0 and
            number % 12 == 0 and
            number % 13 == 0 and
            number % 14 == 0 and
            number % 15 == 0 and
            number % 16 == 0 and
            number % 17 == 0 and
            number % 18 == 0 and
                number % 19 == 0):
            return number


print(True if solution_5() == 232792560 else False)

File: go/solutions/problem5.go
package solutions

// Solution5 => void -> string
func Solution5() string {
	var result int
	number := 10
	runLoop := true
	for runLoop {
		if isDivisible(number) {
			result = number
			runLoop = false
		}
		number++
	}
	if result == 232792560 {
		return "problem 5 solved"
	}
	return "problem 5 not solved"
}

func isDivisible(checkThis int) bool {
	for index := 11; index < 21; index++ {
		if checkThis%index != 0 {
			return false
		}
	}
	return true
}
